Source/CarthageKit/Archive.swift

```diff
@@ -50,6 +52,18 @@ private func untargz(archive fileURL: URL, to destinationDirectoryURL: URL) -> S
 		.then(SignalProducer<(), CarthageError>.empty)
 }
 
+/// Untars the bzipped archive at the given file URL, extracting into the given
+/// directory URL (which must already exist).
+private func untarbzip(archive fileURL: URL, to destinationDirectoryURL: URL) -> SignalProducer<(), CarthageError> {
+	precondition(fileURL.isFileURL)
+	precondition(destinationDirectoryURL.isFileURL)
+
+	let task = Task("/usr/bin/env", arguments: [ "tar", "-xjf", fileURL.path, "-C", destinationDirectoryURL.path ])
```

Comment:
`man tar` on my machine says:

> ```
>      -j      (c mode only) Compress the resulting archive with bzip2(1).  In extract or
>             list modes, this option is ignored.  Note that, unlike other tar implementa-
>             tions, this implementation recognizes bzip2 compression automatically when
>             reading archives.
> ```

It says the same thing about `-z`. So It think we're safe to use the same method for both and drop the `-j` and `-z`.